addons/website_sale/views/templates.xml

```diff
@@ -1167,7 +1167,7 @@
     </template>
 
     <template id="product_tags" name="Product Tags" active="True">
-        <div class="o_product_tags o_field_tags d-flex flex-wrap align-items-center gap-2">
+        <div class="o_product_tags o_field_tags d-flex flex-wrap align-items-center gap-2 mt-2">
```

Comment:
still seems problematic now if it is in other place where mt is not needed then it'll cause misalignment
You should add span where this template is called and add mt there